modules/monitoring/templates/govuk-uptime-collector.conf.erb

```diff
@@ -0,0 +1,9 @@
+description "govuk-uptime-collector"
+
+start on runlevel [2345]
+respawn
+
+env RBENV_VERSION=2.3
+env PATH=/usr/lib/rbenv/shims:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+exec govuk_uptime_collector <%= @environment %> publishing-api content-store
```

Comment:
Out of my own ignorance, do we know what happens if this fails? are we notified at all - or does something monitor this (monitor inception)

Reply:
If it fails, the service will be respawned automatically. I figured we would notice if the uptime stopped being reported, but perhaps we should add something more explicit here?

Reply:
Sure, I'm not sure we would notice - can't imagine we'd look at it regularly guessing someone will see it in weeks or months rather than hourly.

Reply:
I've added an Icinga check for it.

Reply:
Great! will that link to the docs if there's a problem?